api/kessel/relations/v1beta1/common.proto

```diff
@@ -49,3 +49,8 @@ message ObjectType {
 	string namespace = 1 [(buf.validate.field).string.min_len = 1];
 	string name = 2 [(buf.validate.field).string.min_len = 1];
 }
+
+// The Zookie is used to provide consistency between write and read requests.
+message Zookie {
+	string token = 1 [(buf.validate.field).string.min_len = 1];
```

Comment:
Are zookies always of conistent length? if not, a length check of "1" is fine

Reply:
From my own testing with the zookie, no it's not always a consistent length. The length of the token does vary and I've set it up similarly to authzed's docs. They have a token field with a `min_bytes` of 1 which is equivalent to `min_len` of 1. Doc ref: https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.ZedToken

Reply:
Also, the implementation of the zookie is storage-dependent _in SpiceDB_, so even if they were a consistent length for Postgres, they wouldn't necessarily be for, say, CRDB. And another Zanzibar implementation may go a different direction entirely.

min_len = 1 ensures that it's not empty without asserting anything about the contents, which makes sense to me.